pkg/antctl/raw/supportbundle/command.go

```diff
@@ -73,6 +74,8 @@ var remoteControllerExample = strings.Trim(`
   $ antctl supportbundle
   Generate support bundle of the controller
   $ antctl supportbundle --controller-only
+  Generate support bundle of the controller and agents on all Nodes with the last 5 days's logs
```

Comment:
```suggestion
  Generate support bundle of the controller and agents on all Nodes with only the logs generated during the last 5 days
```

Reply:
Thanks for the review. some commits are generated by gihub commits, will be fixed ASAP. The current impl is not accurate through. Say I want last 5 days log, the earliest log file may contains logs that are 6 days ago or 7 days ago, but it's modification time is 5 days ago and it contains some logs we need. So I think may be days is better?

Reply:
i think Antonin's comment here says `logs generated` instead of `log files generated`. So technically it does include the intent where the logs were modified with newly generated log entries within the past 5 days.

Reply:
So do you guys means we need to split the log files if it contains older logs to extract the precise last N days log or use an timestamp filter?